pkg/loghttp/query.go

```diff
@@ -29,8 +31,9 @@ const (
 
 //QueryResponse represents the http json response to a label query
 type QueryResponse struct {
-	Status string            `json:"status"`
-	Data   QueryResponseData `json:"data"`
+	Status     string            `json:"status"`
+	Statistics stats.Result      `json:"statistics"`
```

Comment:
It's not documented on the Prometheus API page, but instead of adding a new field maybe we want to imitate the [stats API](https://github.com/prometheus/prometheus/pull/2408) that @davkal added in Prometheus?

Reply:
I didn't know about this, it's currently a new property at the root of the Jason named statistics.

I'm glad you took a look, let me see how hard would that be to follow the same pattern e.g inside the data property

Reply:
Also I used Nano suffix instead of David's Ns that should be easy to follow